tests/pipeline/nodes/preprocess/test_brightness.py

```diff
@@ -0,0 +1,93 @@
+# Copyright 2021 AI Singapore
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Test for preprocess brightness node
+"""
+
+import numpy as np
+import pytest
+
+from peekingduck.pipeline.nodes.preprocess.brightness import Node
+
+
+@pytest.fixture
+def brightness_same():
+    node = Node({"input": ["img"], "output": ["img"], "beta": 0})
+    return node
+
+
+@pytest.fixture
+def brightness_increase():
+    node = Node({"input": ["img"], "output": ["img"], "beta": 50})
+    return node
+
+
+@pytest.fixture
+def brightness_decrease():
+    node = Node({"input": ["img"], "output": ["img"], "beta": -50})
+    return node
+
+
+class TestBrightness:
+    def test_no_change(self, brightness_same, create_image):
+        original_img = create_image((28, 28, 3))
+        input1 = {"img": original_img}
+        results = brightness_same.run(input1)
+        np.testing.assert_equal(original_img, results["img"])
+
+    def test_brighten_image(self, brightness_increase):
+        original_img = np.ones(shape=(28, 28, 3), dtype=np.uint8)
+        input1 = {"img": original_img}
+        results = brightness_increase.run(input1)
+
+        assert original_img.shape == results["img"].shape
+        np.testing.assert_raises(
```

Comment:
I see a mix of `np.testing` and `import numpy.testing as npt` in our test scripts. Do we want to standardize to one form? And also do we want to prefer `import numpy.testing as npt` since it will make the test code below shorter?

Reply:
I feel that it's fine to be a bit more relaxed about this for unit testing. Some may be very used to the convention of using `numpy` as `np`. If we set too many guidelines it may be hard for apprentices to catch up.